test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -213,6 +216,53 @@ localTest iot nio = do
     assertEqual "expect /local to return gas for tx" (_crGas res) 5
     assertEqual "expect /local to succeed and return 3" e (Right (PLiteral $ LDecimal 3))
 
+localContTest :: IO (Time Micros) -> IO ChainwebNetwork -> TestTree
+localContTest iot nio = testCaseSteps "local continuation test" $ \step -> do
+    cenv <- _getClientEnv <$> nio
+    sid <- mkChainId v maxBound (0 :: Int)
```

Comment:
let's switch to using `unsafeChainId` in tests

Reply:
Fixed